ios/RNScreenshotPrevent.m

```diff
@@ -138,10 +141,7 @@ -(void) removeSecureTextFieldFromView:(UIView *) view {
 
 /** removes secure textfield from the view */
 RCT_EXPORT_METHOD(disableSecureView) {
-    /*UIView *view = [UIApplication sharedApplication].keyWindow.rootViewController.view;
-    for(UIView *subview in view.subviews){
-        [self removeSecureTextFieldFromView:subview];
-    }*/
+    secureField.secureTextEntry = false;
```

Comment:
I think the loop is still needed because there is a condition where this method can be prone to bugs.

1. Call enableSecureView 2 times.
2. Call disableSecureView
3. The screenshot still blank.

CMIIW, I'm not fully understand swift

@killserver @BavlyAbdelmasih